pkg/params/run.go

```diff
@@ -89,7 +89,7 @@ func (r *Run) UpdatePACInfo(ctx context.Context) error {
 		return err
 	}
 
-	if r.Info.Pac.Settings.TektonDashboardURL != r.Clients.ConsoleUI.URL() {
+	if r.Info.Pac.Settings.TektonDashboardURL != "" && r.Info.Pac.Settings.TektonDashboardURL != r.Clients.ConsoleUI.URL() {
```

Comment:
any chance you can explain, how did you fix it ? what was the bug ? good commit message is helpful that just "validation" :)

Reply:
yeah sorry
update the PR desc